### PR TITLE
Read organization members

### LINKED ISSUE #90: Read organization members
Some Conform PR checks will be different between PRs from the community and FerretDB organization members. We should read organization members and check if the PR author belongs to it.

### PR DESCRIPTION
**Pull request by AlekSi**
Closes #90.

### CODE REVIEW

**Review by chilagrow — DISMISSED** [review]
looks good to me.

File: conform-pr/main.go [hunk 1]
```diff
@@ -164,28 +188,38 @@ func checkSize(_ *githubactions.Action, projectFields map[string]graphql.Fields)
...
+	if got != nil {
+		return fmt.Errorf(`PR should have "Size" field unset, got %s.`, strings.Join(got, ", "))
+	}
+
 	return nil
 }
 
-// checkSprint checks that PR has a "Sprint" field set.
-func checkSprint(_ *githubactions.Action, projectFields map[string]graphql.Fields) error {
+// checkSprint checks that PR has a "Sprint" field set for non-community PRs.
```
Comment (rumyantseva) [inline conform-pr/main.go:205]: I don't understand the godoc comment. It says that the check is for non-community PRs, but it actually checks it for all the PRs, not only for non-community PRs.

File: conform-pr/main.go [hunk 2]
```diff
@@ -87,9 +93,33 @@ type checkResult struct {
 	err   error
 }
 
-// runChecks runs all the checks on the given PR GraphQL node.
-func runChecks(ctx context.Context, action *githubactions.Action, client *graphql.Client, nodeID string) []checkResult {
-	pr := client.GetPullRequest(ctx, nodeID)
+// runChecks runs all the checks for the given PR.
```
Comment (rumyantseva) [inline conform-pr/main.go:96]: Let's document what it returns?

Feel freee to rephrase:
```suggestion
// runChecks runs all the checks for the given PR.
// It returns a slice of check's results and true if the PR is created by community member (false if the PR is created by a team member).
```

File: conform-pr/main.go [hunk 3]
```diff
@@ -87,9 +93,33 @@ type checkResult struct {
...
+//
+//nolint:lll // package and type names are long
+func runChecks(ctx context.Context, action *githubactions.Action, client *github.Client, gClient *graphql.Client, org, user, nodeID string) ([]checkResult, bool) {
```
Comment (rumyantseva) [inline conform-pr/main.go:99]: I'm a bit skeptical about the new signature - that second result parameter doesn't look super relevant to what the function is doing. I understand the idea - as we pass org and user to the function, we can identify if the user is a part of the org, and some checks might depend on it, but still it's a little bit outside of the service of this function. I don't have a better idea though, so I don't make this a change request.

**Review by rumyantseva — COMMENTED** [review]
The code looks correct to me. 

I'm unsure about some design decisions (see my comments), but I don't insist on making changes. Please ack my comments, and I'll be ready to approve.

**Review by w84thesun — APPROVED** [review]
LGTM

File: conform-pr/main.go (hunk 3 above)
Comment (noisersup) [inline conform-pr/main.go:99]: Could we split that to multiple lines? WDYT?

**Review by noisersup — APPROVED** [review]
LGTM

File: conform-pr/main.go (hunk 1 above)
Comment (AlekSi) [reply conform-pr/main.go]: Done

File: conform-pr/main.go (hunk 2 above)
Comment (AlekSi) [reply conform-pr/main.go:109]: Done

File: conform-pr/main.go (hunk 3 above)
Comment (AlekSi) [reply conform-pr/main.go]: Another aspect of that – it makes testing easier

Comment (AlekSi) [reply conform-pr/main.go]: Fixed in a different way

**Pull request merged**